smack-integration-test/src/main/java/org/jivesoftware/smackx/muc/AbstractMultiUserChatIntegrationTest.java

```diff
@@ -90,15 +90,22 @@ static void tryDestroy(final MultiUserChat muc) throws SmackException.NoResponse
         muc.destroy("test fixture teardown", null);
     }
 
-    static void createMuc(MultiUserChat muc, String resourceName) throws SmackException.NoResponseException, XMPPException.XMPPErrorException, InterruptedException, MultiUserChatException.MucAlreadyJoinedException, SmackException.NotConnectedException, MultiUserChatException.MissingMucCreationAcknowledgeException, MultiUserChatException.NotAMucServiceException, XmppStringprepException {
-        MultiUserChat.MucCreateConfigFormHandle handle = muc.create(Resourcepart.from(resourceName));
+    static void createMuc(MultiUserChat muc, Resourcepart resourceName) throws SmackException.NoResponseException, XMPPException.XMPPErrorException, InterruptedException, MultiUserChatException.MucAlreadyJoinedException, SmackException.NotConnectedException, MultiUserChatException.MissingMucCreationAcknowledgeException, MultiUserChatException.NotAMucServiceException {
```

Comment:
Even though the line become shorter, it is still very long. Please break it at 120 chars.